cogs/music.py

```
@@ -670,6 +675,15 @@ async def ensure_voice(self, interaction: discord.Interaction) -> mafic.Player |
             saved_volume = self.bot.state_manager.get("volume", default_vol)
             await player.set_volume(saved_volume)
 
+            try:
+                saved_filter = self.bot.state_manager.get("filter")
+                if saved_filter:
+                    filter_obj = get_filter(saved_filter)
+                    if filter_obj:
+                        await player.add_filter(filter_obj, label=FILTER_LABEL)
+            except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)